atlasdb-dbkvs-hikari/src/main/java/com/palantir/nexus/db/pool/config/ConnectionConfig.java

```diff
@@ -79,7 +79,7 @@ public int getMaxConnections() {
 
     @Value.Default
     public Integer getMaxConnectionAge() {
-        return 1800;
+        return 888;
```

Comment:
ideally we'd write this as

```
return SERVER_SIDE_EXPECTED_TIMEOUT - A_FEW_SECONDS
```

so someone understands how the number was achieved

Reply:
I would prefer a comment over adding more constants. Breaking it out into multiple variables makes it harder for someone looking at the code to immediately see what the actual value is, and you probably would want a comment anyway so that people don't increase it later thinking that this is safe.

The important thing which is not obvious is that hikari does not enforce the idle timeout (at all) when the number of idle connections is equal or less than the min pool size. Also when a connection is returned to the pool, hikari does not know how long the connection was idle prior to being returned to the pool. For this reason, this field is effectively the actual idle timeout.